libcalico-go/lib/backend/syncersv1/updateprocessors/configurationprocessor_test.go

```diff
@@ -77,7 +77,7 @@ var _ = Describe("Test the generic configuration update processor and the concre
 		"RouteRefreshInterval":               nil,
 		"IptablesRefreshInterval":            nil,
 		"IpsetsRefreshInterval":              nil,
-		"IpInIpEnabled":                      nil,
+		"DeprecatedIpInIpEnabled":            nil,
```

Comment:
Remnants of the "Deprecated" naming?

Reply:
Yes, I must have missed this and the one below, thanks for catching it!